### PR TITLE
Add examples for striim compatible transform

### PR DESCRIPTION
**Pull request by nishantwrp**
[Striim](https://www.striim.com) is a distributed data integration and intelligence platform that can be used to design, deploy, and run data movement and data streaming pipelines.

This pull request adds  simple steps to set up a YugabyteDB CDC pipeline with `StriimCompatible` transfomer that transforms the messages being sent to kafka in a format compatible with Striim's [WAEvent](https://www.striim.com/docs/en/postgresql-cdc.html#UUID-bb4c7685-acc1-6f6a-f4a2-afbfecf4febf).

### CODE REVIEW

File: striim-transform/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+[Striim](https://www.striim.com) is a distributed data integration and intelligence platform that can be used to design, deploy, and run data movement and data streaming pipelines.
+
+This repository contains simple steps to set up a YugabyteDB CDC pipeline with `StriimCompatible` transfomer that transforms the messages being sent to kafka in a format compatible with Striim's [WAEvent](https://www.striim.com/docs/en/postgresql-cdc.html#UUID-bb4c7685-acc1-6f6a-f4a2-afbfecf4febf).
+
+## Setup
+
+To use `StriimCompatible` transform with the connectors the following properties needs to be added to the connector configuration.
+
+```json
+"transforms": "unwrap",
```
Comment (fourpointfour) [inline striim-transform/README.md:12]: `unwrap` is just a name we give to the transformer. However, in this case, the name seems a little misleading. Maybe we can rename this to `consistent` or `consistency` or `striim`.

File: striim-transform/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+"transforms.unwrap.type": "io.debezium.connector.yugabytedb.transforms.StriimCompatible",
+```
+
+**NOTE**: For this transform to work correctly you need to enable before image while creating the cdc stream. You can do it using the following command.
```
Comment (fourpointfour) [inline striim-transform/README.md:16]: I am guessing this is because of the fact that the transformer needs the full image of the record?

**Review by fourpointfour — APPROVED** [review]
Looks good to me. Just address the renaming comment before merging.

File: striim-transform/README.md (hunk 1 above)
Comment (nishantwrp) [reply striim-transform/README.md:12]: Oh okay, done.

File: striim-transform/README.md (hunk 2 above)
Comment (nishantwrp) [reply striim-transform/README.md:16]: Yes, that's correct.

**Pull request merged**